src/status_im/contexts/profile/settings/screens/password/view.cljs

```diff
@@ -0,0 +1,88 @@
+(ns status-im.contexts.profile.settings.screens.password.view
+  (:require [quo.core :as quo]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.biometric.events :as biometric]
+            [status-im.common.not-implemented :as not-implemented]
+            [status-im.common.standard-authentication.standard-auth.authorize :as authorize]
+            [status-im.constants :as constants]
+            [status-im.contexts.profile.settings.screens.password.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [utils.security.core :as security]))
+
+(defn- on-press-biometric-enable
+  [button-label theme]
+  (fn []
+    (authorize/authorize
+     {:biometric-auth?   false
+      :blur?             true
+      :theme             theme
+      :auth-button-label (i18n/label :t/biometric-enable-button {:bio-type-label button-label})
+      :on-close          (fn [] (rf/dispatch [:standard-auth/reset-login-password]))
+      :on-auth-success   (fn [password]
+                           (rf/dispatch [:hide-bottom-sheet])
+                           (rf/dispatch [:standard-auth/reset-login-password])
+                           (rf/dispatch [:biometric/enable
+                                         (security/mask-data password)]))})))
```

Comment:
@clauxx if you take a look at `on-auth-success` the password is already hashed etc 👍

Reply:
https://github.com/status-im/status-mobile/blob/5fa9c0cab60ea68db4cb156e9595ba5c7a0c5300/src/status_im/common/standard_authentication/standard_auth/authorize.cljs#L21
It depends if the password is from biometrics or password etc 👍

Reply:
I'm just masking it here, so no hashing involved. Maybe it should be masked inside authorize before it's passed to `on-auth-success`, otherwise the whole purpose is kinda lost.